cluster/examples/kubernetes/ceph/pool.yaml

```diff
@@ -30,6 +30,10 @@ spec:
   # Inline compression mode for the data pool
   # Further reference: https://docs.ceph.com/docs/nautilus/rados/configuration/bluestore-config-ref/#inline-compression
   compressionMode: none
```

Comment:
compressionMode and targetSizeRatio in the examples should move under properties, right?